allele
: A version of a gene.

allosteric, allosteric regulation
: Allosteric regulation is one protein binding to another that can increase or decrease a protein's activity.

antisense
: The opposite side of a double-helix strand of DNA.

amino acid
: Amino acids are small molecules used as building blocks for proteins.

bind
: One molecule latching onto another, usually by matching specific complementary shapes and forces on their surfaces.

cerebellum
: Part of the brain at the back of the skull that coordinates and regulates muscular activity.

chromatin
: Chromatin is the material chromosomes are made of.  Proteins, DNA, and RNA.

cleavage
: Cutting

coding, protein coding
: Coding DNA provides instructions for assembling proteins.

codon
: A section of 3 letters of DNA or RNA that is used to specify one amino acid for the ribosome to attach to a protein.  Many adjacent, consecutive codons are used to create a whole protein.

conserved
: DNA that is the same in distantly related organisms.  Under evolutionary theory this indicates function, because otherwise evolution would not have kept them the same.

constructive mutation, constructive
: A "constructive mutation" is defined here as a gain or modification of function that is beneficial.  This is distinguished from loss of function mutations that are beneficial.

cortex
: The outer layer of the cerebrum (front part of the brain) that is involved in consciousness

cytoplasm
: The cytoplasm is the part inside of a cell that is not part of the nucleus (center).

deleterious, deleterious mutation
: In medical science:  A mutation that is harmful.  Or in population genetics: a mutation that reduces the number of offspring an organism produces.

dentate gyrus
: Part of the hippocampus (which is part of the brain) thought to be involved in the development of new memories.

differentiation
: The process where a less specialized cell becomes a more specialized cell.  For example fertilized egg cell dividing and becoming other cell types.

diploid
: Two sets of chromosomes, or an organism that has two sets of chromosomes.

encode
: ENCODE is an ongoing project by the United States National Institute of Health (NIH) to find function of the various sequences of human DNA.  The project involves hundreds of scientists and hundreds of millions of dollars in funding.

enhancer
: A short (50 to 1500 letter) section of DNA where proteins called transcription factors can attach to increase the likelihood that a nearby gene will be used.

epigenome
: Information in a cell outside of the DNA that tells the DNA when and how to be used.

eukaryote, eukaryotic
: A eukaryote is organism with cells that have a nucleus--an enclosed center region that contains the DNA.  Animals, plants, and fungi are some examples of eukaryotes.  Bacteria and archaea are not eukaryotes.

exon
: A section of DNA with information for the order in which to assemble amino acids to make a protein.

express
: Expression is the process of activating and using a gene.

fitness
: The ability of an organism to survive and reproduce.

fix, fixation
: A mutation becomes fixed after a number of generations when it spreads to every surviving member of a population.

ftir
: Fourier-transform infrared spectroscopy: A technique to measure individual colors beyond what the human eye can see.

functional element
: A stretch of DNA that has instructions for creating something within a cell that has a specific function, such as a protein.

genetic diversity
: In a population, different members have different mutations.  Genetic diversity is a measure of the number of differences present in a population.

genetic load
: In simplified terms, genetic load is a measure of the average number of deleterious mutations in a population.

genome
: A genome is all of the genetic material in one of an organism's cells, or in a virus.

haploid
: A single set of unpaired chromosomes, or an organism that has only one set of chromosomes.

hippocampus
: Part of the brain thought to be the center of emotion, memory, and the autonomic nervous system

insulator
: A section of DNA about 300 to 2000 letters long where a protein can attach to prevent genes from being used.

intelligent design
: The idea that some features of the universe and of living things are best explained by an intelligent cause and not undirected processes.

intergenic
: Areas of DNA between protein coding genes.  Protein coding genes are DNA sequences with instructions for assembling proteins.

intron
: A section of DNA that's copied to RNA (along with exons), but is removed from the RNA before the RNA is read by a ribosome to create a protein.

lignin
: Structural support material found in most plants.

lincrna, linc rna
: Long intergenic noncoding RNAs:  A longer RNA that does not go to a ribosome to become a protein, but instead does other functions.

micro rna, microrna, mirna
: A small molecule of about 22 RNA letters that binds to other RNA to specify how it should be used.

mutation
: A change in an organism's genetic material (DNA, or RNA for some viruses).

molecular clock
: Molecular clocks estimate time by counting how many nucleotide (letters) are different between the genetic material of two organisms, and estimating how long it would take for mutations to create those differences.

natural selection, selection
: The process where weaker organisms are less likely to survive and pass on their genes.

nucleic acid
: A string of nucleotides.  DNA and RNA are nucleic acids.

nucleotide
: A nucleotide is a molecule that acts as a "letter" of DNA or RNA

noncoding
: Non-coding is DNA that does not directly provide instructions for assembling proteins.

organelle, organellar
: An organelle is a sub-unit of a cell that performs a specific task.  Mitochondria and ribosomes are examples of organelles. 

pathogenic
: Causing disease.

non-pathogenic
: Not causing disease.

pentose
: A plant sugar with the formula C<sub>5</sub>H<sub>10</sub>O<sub>5</sub>

phenotype, phenotypic
: A phenotype is an organism's observable characteristics.

point mutation
: A point mutation changes one nucleotide  "letter" of DNA to another.

polymorphic
: A nucleotide (letter) of genetic material that's different between two organisms.

promoter
: A section of DNA where a protein latches on and begins copying it to RNA.

protein
: A protein is a molecular machine made in cells by assembling amino acids.

purkinje cells
: Neurons in the cerebellar cortex of the brain.

reproductive rate
: The average number of offspring an organism has.

phosphorylation
: Attaching a phosphorous/oxygen molecule to a protein.

poly(A), poly(A) tail
: A series of adenine (letter A) nucleotides added to the end of an RNA molecule.

polyadenylation, 
: The process of adding a series of adenine (letter A) nucleotides to the end of an RNA molecule.

provirus
: The code of a virus integrated into a host cell.

pyrolysis-mass-spectrometry
: A sample is heated to the point it decomposes to produce smaller molecules that are detected using mass spectrometry to determine what elements make up the sample.

retrovirus
: An RNA virus that inserts its genome into a host cell as DNA.

ribosome
: A ribosome is a factory inside a cell that reads the instructions in RNA to produce a protein.

rna
: In most organisms, RNA is a molecule that stores a copy of the information in DNA to send elsewhere in a cell, and can also perform functions of its own.  Some viruses use RNA instead of DNA as their genetic material.

rna polymerase
: RNA polymerases are protein machines that copy DNA to RNA.

scanning electron microprobe
: A device that bombards a sample with an electron beam, then measures x-rays that bounce back, in order to determine the elements that make up a sample.

shine-dalgarno sequence
: A Shine-Dalgarno sequence is a short section at the start of RNA transcripts from some bacteria, archaea, chloroplasts, and mitochondria.  It's used by ribosomes to start reading the RNA to create a protein. 

soxhlet extractor
: A tool for removing fat/oil from a solid material.

splice, splicing
: Splicing is removing sections DNA or RNA and reconnecting the remaining pieces.

stochastic
: Stochastic means randomly determined.

sturp
: Shroud of Turin Research Project was a team of scientists which performed a set of experiments and analyses on the Shroud of Turin during the late 1970s and early 1980s.

substitution
: A substitution is a mutation that changes one nucleotide "letter" of DNA to another.

substrate
: The natural environment where an organism lives.

tof-sims
: Time-of-flight secondary ion mass spectrometry:  A technique that can be used to detect the chemical makeup of the surface of a sample.

transcribe, transcription, transcriptional
: The process of copying DNA into RNA.

transcript
: RNA that was produced by being copied from DNA.

transfer rna, trna
: Transfer RNA is a molecule of about 76 to 90 RNA letters that is used to bring the correct amino acids to a ribosome so it can assemble them into a protein in the proper sequence.

translate, translation, translated
: Translation happens when a ribosome in a cell reads the instructions on RNA to produce a protein.

transposon, transposable element, jumping gene, mobile element, te
: A transposon (also called jumping genes, mobile elements, or transposable elements) is a piece of DNA that can move or copy itself to different places in the genome.

ultrasonic bath
: A cleaning process that uses low frequency sound to agitate a fluid.

virion
: A complete virus that exists outside of a cell.