<?php
require_once $_SERVER['DOCUMENT_ROOT'].'/init.php';

$title = 'Functional DNA is Evidence of Design | Berean Archive';
$image = '/articles/biology/functional-dna/header-square.jpg';
$headerStyle = "background-image: url('functional-dna/header-wide.jpg')";
$caption = 'The pufferfish[^pufferfish] genome is  8 times smaller than the human genome, a fact 
			used by some to argue that most	human DNA is unused junk.';


// $content
ob_start()?>
<aside>
	Published around May 2017
</aside>

<h1>Functional DNA is Evidence of Design</h1>


<p><q>“It’s likely that 80 percent [estimate of functional human DNA] will go to 100 percent. We
	don’t really have any large chunks of redundant DNA. This metaphor of junk isn’t that useful."
	— <dfn>ENCODE</dfn> lead researcher Ewan Birney.[^yong-2012]</q></p>
<p><q>"If the human <dfn>genome</dfn> is indeed devoid of junk DNA as implied by the ENCODE
	project, then a long, undirected evolutionary process cannot explain the human genome... If
	ENCODE is right, evolution is wrong." — evolutionary biologist and atheist Dan
	Graur[^graur-2013b]</q></p>
<h2>Overview</h2>
<p>Humans and other mammals have about 3 billion "letters" of DNA.&nbsp; If evolution produced
	complex mammals like humans, we should expect most DNA to have no function, because:</p>
<ol>
	<li>3 billion DNA letters is much more function than evolution could produce, given even
		best-case scenarios.[^dodson-1962] [^moran-2014]</li>
	<li>Humans also get about 100 mutations per generation.[^moran-2013] Most
		mutations that have any effect on an organism are harmful.[^lind-2010]&nbsp; If even 10% of
		DNA has a specific sequence, then humans and other mammals would receive almost 10 new
		harmful mutations each generation, causing all offspring to have more harmful mutations
		than their parents.[^ohno-1972] [^gibson-2011] [^moran-2014b] Function would perpetually
		decline with each generation.&nbsp; If evolution can't even preserve the function in
		our DNA, it could not have created it.</li>
</ol>
<p>However if organisms were designed, it would not make sense to create them with
	non-functional DNA.&nbsp; Their only junk DNA would be that which has subsequently been
	destroyed by mutations.</p>
<h3>How much human DNA is in functional elements?</h3>
<aside>
	<iframe style="width: 100%;" src="https://www.youtube.com/embed/5MfSYnItYvg" frameborder="0"
	allowfullscreen="allowfullscreen" mozallowfullscreen="mozallowfullscreen" msallowfullscreen="msallowfullscreen"
	oallowfullscreen="oallowfullscreen" webkitallowfullscreen="webkitallowfullscreen"></iframe>
	This video[^dna-transcription-2010] shows how DNA is copied (transcribed) into RNA.
</aside>
<p>Most human DNA has not yet been tested to see if it has a function.[^mattick-2013]&nbsp; But we
	can still extrapolate from known data:</p>
<ol>
	<li>At least 85% of DNA is copied (transcribed) into
		<dfn>RNA</dfn>.[^hangauer-2013]
		<br>
	</li>
	<li>When and where DNA is copied to RNA occurs in specific patterns that depend on the cell
		type and the stage of development.[^mattick-2010] [^mattick-2013]&nbsp;
		[^hangauer-2013]</li>
	<li>Among DNA copied to RNA transcripts in the human brain, at least 80% are taken to specific
		locations within their cells.[^mattick-2010]<br></li>
	<li>Enough RNA has been tested for function that we can "draw broader conclusions about
		the likely functionality of the rest."[^mattick-2013]</li>
</ol>
<p>If at least 85% of DNA is copied to RNA, and at least 80% of those RNAs are taken to specific
	locations within cells, 85% * 80% = at least 68% of human DNA is used in a functional way.
	&nbsp;And likely much more because these are both lower-bound estimates.&nbsp; As function
	continues to increase as more DNA is studied, it is reasonable to think that perhaps even 99%+
	DNA is in use.[^yong-2012] This is not to say that every DNA letter within these sequences
	requires a specific sequence.</p>
<h3>How much human DNA requires a specific sequence of letters?</h3>
<p>Within a functional element of DNA, not all letters have to have a specific sequence. But how
	many do?</p>
<ol>
	<li>If 85% of DNA is copied to RNA transcripts, 80% of those RNA transcripts are functional
		(transported to specific locations),[^mattick-2010]  and 66% of that RNA
		requires a specific sequence[^parker-2011], then these three numbers
		multiplied suggests at least 45% of DNA requires a specific sequence.</li>
	<li>At least 20% of DNA consists of either specific sequences where <dfn>proteins</dfn>
		<dfn>bind</dfn> to it, or instructions for making proteins (exons)[^encode-2012], and much
		known function that exists outside of protein binding spots and exons.
	</li>
	<li>About 95% of mutations that cause noticeable effects are outside of the 1-3% of DNA that
		creates proteins.[^maurano-2012] [^freedman-2011]&nbsp; From this (calculated below) we
		can extrapolate that at least 30% of DNA has a specific sequence.</li>
	<li>About 16% to 30% of either human DNA, or RNA copied from it, is shared (conserved) with distantly
		related mammals.[^smith-2013]&nbsp; Distant enough that if we assume they evolved from a
		common ancestor, then mutations would have had enough time to scramble this DNA if it were
		non-functional.</li>
</ol>
<p>Since these are each under-estimates, and partially non-overlapping, the true amount of
	sequence-specific DNA is likely greater.<br></p>
<h3>Implications</h3>
<p>The data on the amount of functional DNA nullifies the argument that almost all DNA is junk
	and is therefore not designed.</p>
<p>Furthermore, at least 16-45% of DNA has a specific sequence, and likely much more.&nbsp; This
	implies humans get at least 16 to 45 harmful mutations per generation--far too many for
	evolution to even prevent declining function.</p>
<div class="highlight">
	Evidence is what is expected in one view and not expected in competing
	views. Therefore the large amounts of functional DNA is evidence that organisms were designed.</div>
<p>The following sections outline this data in greater detail.</p>
<h2>Defining Terminology</h2>
<aside><img src="/common/img/portraits/michael-behe.jpg?w=230" class="rounded" style="width: 100%;">
	<p>Examples of functional elements as defined by Michael Behe:[^behe-2010]<br></p>
	<ul class="compact">
		<li><dfn>promoters</dfn></li>
		<li>enhancers</li>
		<li>insulators</li>
		<li>Shine-Dalgarno sequences</li>
		<li>transfer RNA genes</li>
		<li>micro RNA genes</li>
		<li>protein coding sequences</li>
		<li>organellar targeting- or localization-signals</li>
		<li>intron/extron splice sites</li>
		<li>codons specifying the binding site of a protein for another molecule (such as
			its substrate, another protein, or a small allosteric regulator)</li>
		<li>codons specifying a processing site of a protein (such as a cleavage,
			myristoylation, or phosphorylation site)</li>
		<li>polyadenylation signals</li>
		<li>transcription and translation termination signals</li>
	</ul>
	<p>Here, Michael Behe is merely formalizing an already widely accepted definition of
		functional elements.<br></p>
</aside>
<p>Rather than using only two categories of function and junk, this article takes a more nuanced
	approach:</p>
<ol class="compact">
	<li><b>Sequence-Specific Function:</b>&nbsp; These are nucleotides where a
		<dfn>substitution</dfn> mutation will alter the function or efficiency of a
		functional element, even if only very slightly. Sequence-specific function is a subset of
		the DNA that is in functional elements.</li>
	<li><b>Functional Elements:</b>&nbsp; Functional elements are sequences of DNA that
		perform a specific function.&nbsp; Some of their <dfn>nucleotides</dfn> are
		sequence-specific functional while others can be swapped with no detectible effect on
		function.&nbsp; Or in more technical terms:&nbsp; a sequence of nucleotides that "affects
		the production, processing, or biological activity of a particular <dfn>nucleic
		acid</dfn> or <dfn>protein</dfn>, or the <dfn>binding</dfn> of other
		molecules."[^behe-2010]</li>
	<li><b>Junk DNA:</b>&nbsp; These nucleotides can be removed without degrading
		any functional elements.</li>
</ol>
<p>Other definitions are often used in the debate over how much DNA is functional, but most
	are less precise. Because most genome studies focus on the human genome this article
	does same, as there is the most data is available there.</p>
<h2>Counting DNA in functional elements<br></h2>
<p>As defined above, functional elements have DNA where substitution mutations may or may not
	degrade function, but their removal would break function.  Although "most elements in the human
	genome have not been subject to functional analysis,"[^mattick-2013] we have four reasons to
	think the majority of human DNA is part of functional elements:</p>
<h3>1.&nbsp; More than 85% of human DNA is copied to RNA</h3>
<p>DNA is transcribed when it is copied to RNA.&nbsp; In 2013 genome researchers noted:</p>
<blockquote>We found evidence that 85.2% of the genome is transcribed. This result closely agrees
	with [ENCODE's estimate of] <dfn>transcription</dfn> of 83.7% of the genome...&nbsp; we observe
	an increase in genomic coverage at each lower read threshold implying that even more read depth
	may reveal yet higher genomic coverage.[^hangauer-2013]</blockquote>
<p>ENCODE is an ongoing project by the United States National Institute of Health (NIH) to find
	function of the various sequences of human DNA.&nbsp; The project involves hundreds of
	scientists and hundreds of millions in funding.&nbsp; Prior to the more recent estimate
	of 85.2% transcription, ENCODE found in 2012:</p>
<blockquote>The vast majority (80.4%) of the human genome participates in at least one biochemical
	RNA- and/or <dfn>chromatin</dfn>-associated event in at least one cell
	type.[^encode-2012]
	<br>
</blockquote>
<p>The following chart[^kellis-2014] summarizes the results of various functional tests performed
	by ENCODE in 2012:</p>
<aside>
	<p><b>Transcripts:</b>&nbsp; Columns 1-3.&nbsp; DNA that is copied into RNA</p>
	<p><b>Binding:</b><span>&nbsp; Columns 4-6.&nbsp; Regions of DNA that a protein binds
		to.</span></p>
	<p><span><b>Histone Marks:</b>&nbsp; Columns 7-14. Histones are proteins that DNA is
		wound around within chromosomes. Histone marks are modifications that affect when DNA
		is copied to RNA.</span></p>
</aside>
<p><img src="functional-dna/encode-percent-of-function2.jpg" width="100%" class="inverted"></p>
<p></p>
<p>The third bar above (all RNA) shows that in 2012, at least 75% of DNA was found being copied to
	RNA. The tan regions in the first three bars corresponds to regions of DNA where about "one
	transcript copy per cell"[^kellis-2014] was detected. Bars four through fourteen show the
	amount of DNA seen to be participating in other types of activity that implies function.</p>
<p>Many tests for function do not overlap other tests.&nbsp; This chart[^kellis-2014]
	illustrates the intersections of some of the tests.&nbsp; Note that almost all DNA is colored by
	at least one test of function:</p>
<table class="small" style="width: 100%;">
	<tbody>
		<tr>
			<td><img src="functional-dna/encode-percent-of-function1.png" width="352" height="253" class="inverted">
			</td>
			<td>
				<p><b>Biochemical evidence:</b>&nbsp; These blue regions show DNA that
					participates in the activities in the previous chart.</p>
				<p><b>Genetic evidence:</b>&nbsp; This green region shows DNA that causes
					a noticeable change in function if modified.</p>
				<p><b>Evolutionary evidence:</b>&nbsp; This red circle shows DNA that is
					the same in both humans and some distantly related mammals.</p>
				<p><b>Protein-coding:</b>&nbsp; This purple circle shows DNA that is
					directly used to make <dfn>proteins</dfn>.</p>
			</td>
		</tr>
	</tbody>
</table>
<p><br></p>
<p>However, ENCODE critics often counter-argue that DNA being copied to RNA and
	participating in the other biological activities is not enough evidence that DNA is functional.
	&nbsp;Genomicist Dan Graur is among these critics:</p>
<blockquote>ENCODE ignores the fact that <dfn>transcription</dfn> is fundamentally a
	<dfn>stochastic</dfn> process. Some studies even indicate that 90% of the <dfn>transcripts</dfn>
	generated by <dfn>RNA polymerase</dfn> II may represent transcriptional noise. In fact, many
	transcripts generated by transcriptional noise exhibit extensive association with
	<dfn>ribosomes</dfn> and some are even <dfn>translated<dfn>.[^graur-2013]</blockquote>
<p>Dan Graur goes on to discuss why ENCODE's tests for histone modification, open chromatin,
	transcription factor binding, and DNA methylation alone are not necessarily indicators of
	function.&nbsp; Fair enough.&nbsp; But several additional evidences suggest transcribed DNA is
	likely functional:</p>
<h3>2.&nbsp; DNA is copied to RNA in specific patterns</h3>
<p>Human development involves a single fertalized egg cell dividing into trillions of cells to
	eventually produce an adult human.&nbsp; These resulting cells come in many different types such
	as the various types of bone, muscle, and nerve cells.&nbsp; We see different cell types use
	different sections of DNA at different stages of development, in precise and reproducible
	patterns.</p>
<aside><img src="/common/img/portraits/john-mattick.jpg?w=230" class="rounded">John Mattick is a
	molecular biologist whose research focuses on non-coding DNA.&nbsp; Mattick directs Genomics England
	and is a member of the Australian Academy of Science.
	&nbsp;Mattick is a proponent of evolutionary theory.</aside>
<p>For example, Genome researcher John Mattick describes his observations of DNA being copied
	to RNA in the human brain and elsewhere:</p>
<blockquote>
	<p>Some are only expressed in the <dfn>dentate gyrus</dfn> of the <dfn>hippocampus</dfn>,
		others in particular layers of the <dfn>cortex</dfn>, and others in <dfn>Purkinje
		cells</dfn> in the <dfn>cerebellum</dfn>.[^mattick-2010]</p>
</blockquote>
<blockquote>
	<p>[T]he vast majority of the mammalian genome is differentially transcribed in precise
		cell-specific patterns to produce large numbers of <dfn>intergenic</dfn>, interlacing,
		<dfn>antisense</dfn> and intronic non-protein-coding RNAs, which show dynamic regulation
		in embryonal development, tissue <dfn>differentiation</dfn> and disease with even regions
		superficially described as "gene deserts" expressing specific transcripts in particular
		cells... Assertions that the observed transcription represents random noise (tacitly or
		explicitly justified by reference to <dfn>stochastic</dfn> ("noisy") firing of known,
		legitimate <dfn>promoters</dfn> in bacteria and yeast), is more opinion than fact and
		difficult to reconcile with the exquisite precision of differential cell- and
		tissue-specific transcription in human cells.[^mattick-2013]</p>
</blockquote>

<p>Other genome researchers have noted the same:</p>

<blockquote>
	<p>[T]he <dfn>lincRNAs</dfn> we identified have many characteristics that are inconsistent with
		noise, including specific regulation of their <dfn>expression</dfn>, the presence of
		<dfn>conserved</dfn> sequence and evidence for regulated processing. Furthermore, these
		lincRNAs are strongly enriched with <dfn>intergenic</dfn> sequences that were previously
		known to be functional in human traits and diseases.[^hangauer-2013]</p>
</blockquote>

<blockquote>Independently, the developmental and tissue-specific expression of most ncRNAs 
provides perhaps the most compelling case for their widespread functionality. A study of ncRNAs 
expressed in mouse brain by in situ hybridization showed that the majority (623 out of 849) are 
selectively expressed in discrete functional regions of the brain, sometimes with evidence of 
specific subcellular localization. Moreover, expression signatures and dynamic regulation of 
hundreds of ncRNAs has been observed across tissue types and in various developmental 
systems, from Drosophila embryogenesis to differentiation of mammalian ES cells, 
T-cells and muscle cells (Mattick is a co-author of this paper)[^dinger-2012]</blockquote>

<p>If "transcription is fundamentally a <dfn>stochastic</dfn> process" as Graur argues, we should
	not expect transcription to be so precisely regulated.</p>



<h3>3.&nbsp; RNA is taken to specific locations</h3>
<p>John Mattick observed that among non-coding RNA produced in the human brain, 80% of it was
	taken to specific locations within cells:</p>
<blockquote>
	<p>[I]n 80% of the cases where we had sufficient resolution to tell, these RNAs [in the human
		brain] are trafficked to specific subcellular locations. So this is not some fuzzy random
		signal: their <dfn>expression</dfn> is extremely precise, both in terms of the cell
		specificity and in terms of subcellular localization.[^mattick-2010]</p>
</blockquote>
<p></p>
<h3>4. RNAs usually affect development or disease</h3>
<p>John Mattick describes his experience:</p>
<blockquote>
	<p>In fact almost every time you functionally test a non-coding RNA that looks interesting
		because it's differentially <dfn>expressed</dfn> in one system or another, you get
		functionally indicative data coming out.[^mattick-2010]</p>
</blockquote>
<blockquote>
	<p>[W]here tested, these noncoding RNAs usually show evidence of biological function in
		different developmental and disease contexts, with, by our estimate, hundreds of validated
		cases already published and many more en route, which is a big enough subset to draw broader
		conclusions about the likely functionality of the rest.[^mattick-2013]</p>
</blockquote>


<p>But is Mattick merely just assuming this DNA is functional without proper investigation?  
Or perhaps mutations are merely activating nonfunctional DNA, causing the "disease contexts?"&nbsp;   No.&nbsp;  Rather, disabling
this DNA causes loss of function, as can be seen in Mattick's numerous citations:</p>

<ol class="compact">
    <li>"<b>Knockdown</b> of lincRNAs has major consequences on gene expression patterns, comparable to knockdown of well-known ES [embryonic stem] cell regulators."[^guttman-2010]</li>
	<li>"Decreased capacity for cell migration was also observed for SPRY4-IT1 <b>knockdown</b>." (although knocking it out also negatively affected melanoma cells)[^khaitan-2011]</li>
	<li>"We identified lncRNAs required for neurogenesis. <b>Knockdown</b> studies indicated that loss of any of these lncRNAs blocked neurogenesis"[^ng-2012]</li>
	<li>"<b>Knockdown</b> of MEN ε/β expression results in the disruption of nuclear paraspeckles."[^sunwoo-2009]</li>
	<li>"<b>Knockdown</b> of Zfas1 in a mammary epithelial cell line resulted in increased cellular proliferation and differentiation."[^askarian-amiri-2011]</li>
</ol>


<p>It's also notable that Mattick found this much function because "loss-of-function tests can also
	be buffered by functional redundancy, such that double or triple disruptions are required for a
	phenotypic consequence."[^kellis-2014] This means that many genes will not
	produce a noticeable effect when disabled, because when one gene is disabled, cells often
	automatically enable other genes to do the same job.</p>


<div class="highlight">
	<h4>Estimating Minimum DNA in Functional Elements<br></h4>
	If at least 85% of DNA is copied to RNA, and at least 80% of RNA transcripts are taken to
	specific locations within a cell, then we could estimate that perhaps 85% * 80% = <b>at
	least 68%</b> of DNA is within functional elements.&nbsp; And likely much more
	because these are both lower-bound estimates, not all DNA must be transcribed to have function,
	and not all RNA must be taken to a specific sub-cellular location to be functional.
</div>
<p><span style="background-color: #ff6600;"></span></p>
<h2>Counting Sequence-Specific DNA</h2>
<p>As defined above, DNA that is sequence-specific has <dfn>nucleotides</dfn> where a
	<dfn>substitution</dfn> mutation will <span>alter the function or efficiency
	of</span> a functional element, even if only very slightly.&nbsp; We
	have several criteria by which to estimate how much DNA is sequence-specific:</p>
<h3>1.&nbsp; From RNA Structure + Functional Elements</h3>
<p>We also know that "the <dfn>nucleic acids</dfn> that make up RNA connect to each other in very
	specific ways, which force RNA molecules to twist and loop into a variety of complicated 3D
	structures."[^garvan-2013]&nbsp; This places many of the nucleotides transcribed
	to RNA in the sequence-specific category.</p>
<p>Below is a diagram[^parker-2011] with 13 examples of various known functional RNAs:</p>
<img src="functional-dna/rna-structure.png" class="wide inverted">
<p><br></p>
<p>This list shows the number of cross-linked RNA nucleotides (connections where the RNA folds back
	on itself) per total nucleotides for each of the functional RNAs:</p>
<table class="small" style="width: 100%; table-layout: fixed;">
	<tbody>
		<tr>
			<td>
				<p>MAT2A:&nbsp; 26 / 41<br>CTLA4:&nbsp; 12 / 15<br>
					<span>FOS:&nbsp; 12 / 15 <br>
					</span>GRIA1:&nbsp; 26 / 34<br>
					CACNAD2D1:&nbsp; 22 / 37
				</p>
			</td>
			<td>
				<p>COL5A2:&nbsp; 14 / 21<br>UBE2W:&nbsp; 14/ 18<br>
					<span>CLCN5:&nbsp; 28 / 47 <br>
					</span>BCL11B:&nbsp; 20 / 24
				</p>
			</td>
			<td>
				<p>MALAT1:&nbsp; 36 / 56<br>POP1:&nbsp; 40 / 62<br>
					<span>long int. 19931:&nbsp; 60 / 86 <br>
					</span>long int. 16685:&nbsp; 62 / 110
				</p>
			</td>
		</tr>
	</tbody>
</table>
<p><br></p>
<p>Adding these gives 372 cross-linked nucleotides out of 566 total nucleotides, or 66%.&nbsp; From
	this we can make a lower-bound estimate of the percentage of nucleotides in functional RNAs that
	are sequence-specific:&nbsp; Since cross-linked RNA nucleotides require a specific
	sequence, this means at least 66% of the nucleotides within functional RNAs are
	sequence-specific.&nbsp; Although some of the others will be specific also.</p>
<p>Above we estimated at least 68% of DNA is within functional elements.&nbsp; Multiplying
	this number by the 66% give us at least <b>45%</b> of DNA being sequence-specific.&nbsp; This should
	be treated as only a very rough estimate; a placeholder until better data is available.</p>
<h4>Conservation bias?</h4>
<p>The diagram above comes from a study looking at conserved DNA to find functional RNAs.
	&nbsp;Some may argue this introduces a bias, because RNAs with less specific
	sequences would be less likely to show up in a conservation study. However compare those
	above to a functional RNA such as MiRNA-95, where "its conservation is limited to the primate
	lineage and a few other higher mammals"[^frankel-2014] but has 60 out of 79[^hsa-mir-95] (76%)
	cross-linked RNA nucleotides.</p>
<h3>2.&nbsp; From Exons + Protein Binding</h3>
<p><span><dfn>ENCODE</dfn> estimated in 2012 that 20% of the genome is made of exons or has a
	specific sequence that proteins use to determine where to latch on to DNA:</span></p>
<blockquote>[E]ven with our most conservative estimate of functional elements (8.5% of putative
	DNA/protein binding regions) and assuming that we have already sampled half of the elements from
	our transcription factor and cell-type diversity, one would estimate that at a minimum 20% (17%
	from protein binding and 2.9% protein coding gene exons) of the genome participates in these
	specific functions, with the likely figure significantly higher.[^encode-2012]</blockquote>
<p>Because this paper previously mentions primate-specific DNA, some critics claim the 20%
	only applies to the regions of human DNA shared with other primates. However, leading
	ENCODE researcher Ewan Birney references the 20% in regard to the entire human genome:</p>
<blockquote>
	A conservative estimate of our expected coverage of exons + specific DNA:protein
	contacts gives us 18%, easily further justified (given our sampling) to 20%.[^birney-2012]<br></blockquote>
<h3>3.&nbsp; From Disease + Trait Association</h3>

<p>A review in 2012 looked at 920 studies involving the genomes of thousands of
	people[^maurano-2012].&nbsp; They found that only 4.9% of function altering mutations occurred
	within protein-coding DNA (red slice in figure B):</p>
<p><img src="functional-dna/maurano-2012-gwas.png" class="wide inverted"></p>
<p><span>Likewise another broad research review found only 4% of function altering
	mutations (105/2593) were within coding regions.[^freedman-2011]</span></p>
<p>From this we can estimate what percentage of DNA is subject to deleterious mutations.</p>
<h4>Estimating the percentage of deleterious mutations within exons</h4>
<p>Within coding regions, a 2011 study[^freedman-2011] found 79 non-synonymous and 26
	synonymous function altering mutations.&nbsp; About 30% of protein coding mutations are
	synonymous.&nbsp; If synonymous nucleotides were as functional as non-synonymous nucleotides we
	should expect 30%*(26+79) = 31.5 of them to be function-altering.&nbsp; But only 26 of them are
	functional, suggesting that they are only 26/31.5=83% as functional as non-synonymous sites.</p>
<p>At least 75%[^axe-2004] of amino-acid altering mutations within exons are deleterious, which
	implies that 75%*83% =62.25% of mutations at synonymous sites are deleterious.&nbsp;  Combining
	the synonymous and non-synonymous rates gives 75% * 70% + 62% * 30% = 71.1% of mutations within
	exons being deleterious.&nbsp; Although one bacterial study estimated that as many as 95%
	of mutations within exons are deleterious.[^lind-2010]</p>
<h4>Estimating functional DNA</h4>
<p>If 2% of DNA is protein coding, and only 4.9% of harmful mutations fall within protein coding
	DNA, that means non-coding DNA is 2% / 4.9% = 41%&nbsp; as specific in sequence as coding DNA.
	&nbsp;Coding DNA is 71% sequene specific, so about 71% * 41% = 29% of non-coding DNA nucleotides
	are sequence-specific.</p>
<p>Putting it all together:&nbsp; 2% * 71% + 98% * 29% = 30%.&nbsp; That means GWAS studies suggest
	at least <b>30%</b> of DNA is sequence-specific.</p>
<p>This is an under-estimate because mutations within non-coding DNA likely to have a smaller
	effect than mutations in coding DNA, and are thus harder to detect.[^mattick-2009]&nbsp; Thus
	It is likely that even fewer than 4-5% of function altering mutations are within coding DNA.</p>
<h4>Are genome wide association studies unreliable?<br></h4>
<p>Some argue these numbers are invalid because genome-wide association studies
	may be showing false positives due to statistical noise. However, the mutations identified as
	function altering in the 2012 study above usually map to known regulatory features:</p>
<blockquote>
	<p>Fully 76.6% of all noncoding GWAS SNPs either lie within a DHS or are in complete linkage
		disequilibrium with SNPs in a nearby DHS.[^maurano-2012]</p>
</blockquote>
<p>Other researchers have noted the same:</p>
<blockquote>
	<p><span>Of the 9184 GASs, 8733 (95.09%) and 5853 (63.73%) were mapped to at least one
		regulatory features, including promoter or enhancer, regulatory motifs, DNase footprinting
		sites, expression quantitative trait loci (eQTL) and conserved sequences, using the
		HaploReg&nbsp; and RegulomeDBdatabases, respectively. Moreover, many of these GASs 
		were predicted to be associated with
		multiple regulatory features, suggesting their dynamic regulatory
		functions.[^chen-2015]
	</p>
</blockquote>
<p></p>
<h4>Why do only a tiny fraction of nucleotides map to function discovered in genome-wide
	association studies?<br></h4>
<ol>
	<li>Human genomes are highly identical.&nbsp; Genome-wide association studies can only
		probe function where variation exists, and not enough genomes are yet available.</li>
	<li><span>Where variation does exist, multiple people with the same variation must be surveyed
		to rule out effects caused by environment, diet, or exercise.</span></li>
	<li>Most known traits have not yet been tested in genome-wide association studies.</li>
	<li>Many non-coding alleles exist in high copy numbers.  A mutation disrupting only
	one of them would likely not show a phenotypic effect that would show up in a GWAS.</li>
</ol>
<p>As ENCODE researchers explain:&nbsp; "At present, significantly associated loci explain only a
	small fraction of the estimated trait heritability, suggesting that a vast number of additional
	loci with smaller effects remain to be discovered"[^kellis-2014]</p>
<h3>4.&nbsp; From Conserved DNA</h3>
<p>Assuming common descent is true:&nbsp; If two distantly related organisms have some of the same
	DNA, it is likely functional.&nbsp; Because otherwise over tens of millions of years, random
	mutations in each species would have changed the DNA enough that it is no longer
	recognizable as similar.&nbsp; In other words:&nbsp; whatever mutated that DNA didn't survive.
	&nbsp;Such DNA is called conserved DNA, or DNA that is under selection.</p>
<p>However, if organisms are designed and common descent is false, having sequences of DNA in
	common could also imply common design and therefore function.&nbsp; But without the
	assumption of common ancestry, shared sequences between not just distantly related, but all
	organisms would be used.&nbsp; Thus even the 95-96% of DNA shared between humans and
	chimpanzees[^preuss-2012] would imply that 95-96% of DNA is sequence-specific.</p>
<p><span>In 2014, ENCODE researchers estimated that <b>16 to 26%</b> of DNA
	(conserved+lineage specific) is preserved by natural selection:</span></p>
<blockquote>
	<p>[E]stimates that incorporate alternate references, shape-based constraint, evolutionary
		turnover, or lineage-specific constraint each suggests roughly two to three times more
		constraint than previously (<b>12–15%</b>), and their union might be even larger
		as they each correct different aspects of alignment-based excess constraint... Although
		still weakly powered, human population studies suggest that an additional <b>4 -
		11%</b> of the genome may be under lineage-specific constraint after specifically
		excluding protein-coding regions, and these numbers may also increase as our ability to
		detect human constraint increases with additional human genomes...[^kellis-2014]</p>
</blockquote>
<p>Another study estimated at least <b>15.6-30%</b> of DNA is conserved when DNA's
	resulting RNA structure is also taken into account:</p>
<blockquote>
	<p><span>Our findings provide an additional layer of support for previous reports advancing
		that &gt;20% of the human genome is subjected to evolutionary selection...</span></p>
	<p><span><span>the RNA structure predictions we report using conservative thresholds are likely
		to span</span><b> &gt;13.6%</b>
		<span>of the human genome we report.&nbsp; This number is probably a substantial
		underestimate... A less conservative estimate would place this ratio
		somewhere</span><b> above 20%</b>
		<span>from the reported sensitivities measured from native RFAM alignments
		and </span><b>over 30%</b>
		<span>from the observed sensitivities derived from sequence-based realignment of RFAM
		data...</span>
		</span></p>
	<p>the majority (87.8%) of the ECS predictions reported herein lie outside
		annotated sequence-constrained elements...</p>
	<p>we can postulate a revised lower bound of functional sequence in the human
		genome at ~<b>15.6%</b>.[^smith-2013]</p>
</blockquote>
<p>Some studies of conserved mammal DNA estimate lower amounts, but they do not take shape and RNA
	structure conservation into account,[^rands-2014] do not include lineage-specific conservation,
	or they were done before more mammal genomes were available for study.</p>
<p>Furthermore, this is not to say that natural selection can actually maintain
	16-30% of the DNA in a mammal.&nbsp; With this much function, harmful mutations would arrive
	faster than natural selection can remove them, as is described below in the Genetic
	Load section.</p>
<div class="highlight">
	<h4>Estimating Minimal Sequence-Specific Function</h4>
	<p>The above sections use four different criteria to very roughly estimate the lower-bound amount of
		sequence-specific DNA:</p>
	<ol class="compact">
		<li>From RNA structure + DNA in functional elements:&nbsp; At least 45%</li>
		<li>From protein binding + exons:&nbsp; At least 20%</li>
		<li>From disease + trait association:&nbsp; At least 30%</li>
		<li>From conservation:&nbsp; At least 16 to 30%</li>
	</ol>
	<p>Since these are each under-estimates, and partially non-overlapping, the true amount of
		sequence-specific DNA is likely greater.&nbsp; It is therefore very unlikely that all
		four are wrong and the amount of sequence-specific DNA is less than these numbers.</p>
</div>
<h2>Evolutionary Theory Predicted and Requires Junk DNA</h2>
<p><dfn>Intelligent Design</dfn> (ID) proponents predicted that most DNA would turn out to be
	functional.&nbsp; However evolutionary theory both predicts and requires that most DNA in
	complex organisms (with large <dfn>genomes</dfn> and low <dfn>reproductive rates</dfn>) will be
	junk.&nbsp; Two main reasons:</p>
<aside class="companionArticle"><img src="functional-dna-predictions/header-square-small.jpg?w=230"
	class="rounded">

	<p>A companion article, <a href="/articles/biology/functional-dna-predictions/">Predictions of
		Junk versus Functional DNA</a>, goes into greater detail on the rationale and history of
		the predictions of junk DNA, the predictions of function made by intelligent design
		proponents, and catalogs prominent biologists who changed their minds.</p>
</aside>
<h3>Neutral Theory</h3>
<p>The neutral theory of evolution claims that most of the DNA differences between
	complex organisms is by chance, and not because natural selection favored organisms having
	those mutations.&nbsp; The reasoning is as follows:&nbsp; </p>
<ol>
	<li>In typical populations of complex organisms, natural selection can only help spread or
		remove mutations that have a strong effect.&nbsp; Otherwise, random chance is the dominant
		factor in deciding which organisms reproduce and pass on their genes.</li>
	<li>Since most mutations have very little effect on an organism,<span>[^eyre-walker-2007]
		[^lind-2010]</span> neutral theory is a mathematical reality.[^lynch-2006]
		"[T]he revolution is over. Neutral and nearly neutral theory won,"<span>
		</span> as biologist PZ Myers described.[^myers-2014]</li>
</ol>
<p>In humans and other large mammals, greater than 99.98% of mutations would spread through the
	population by chance rather than because of natural selection.[^dodson-1962] [^moran-2014]</p>
<p>Only about 5 to 10% of DNA is shared between humans and more distantly related mammals such as
	horses, dogs, and mice.[^meader-2010]&nbsp; Therefore if these animals all evolved from a
	common ancestor, given neutral theory, the large majority of DNA not shared by these mammals
	would have come about by chance and not natural selection.&nbsp; Since DNA that exists only
	by chance will have a random sequence, it is therefore highly unlikely to be functional.
	&nbsp;Evolutionary biologist and intelligent design critic Dan Graur explains:</p>
<blockquote><img src="/common/img/portraits/dan-graur.jpg?w=100" class="img-left">
	<p><span><span>...there exists a misconception among functional genomicists that the
		evolutionary process can produce a genome that is mostly
		functional.[^graur-2013]</span></span></p>
	<p><span>If the human genome is indeed devoid of junk DNA as implied by the
		</span><dfn>ENCODE</dfn>
		<span>project, then a long, undirected evolutionary process cannot explain the human
		genome.[^graur-2013b]</span>
	</p>
</blockquote>
<p>This is not to say that mutations in almost all DNA would have no effect.&nbsp; The whole
	purpose of this article is to argue the opposite.</p>
<div class="highlight">
	<h4>Why Neutral Theory Matters</h4>
	<p>If evolutionary theory is true, then most DNA in complex organisms must have no function
		because a highly neutral evolutionary process could create nothing better.
		&nbsp;Therefore large amounts of functional DNA is good evidence against evolutionary
		theory.</p>
</div>
<h3>Genetic load</h3>
<p>Genetic load (also called mutational load) is the average number of
	<dfn>deleterious</dfn> (harmful) mutations per organism in a
	population.&nbsp; If the genetic load is too high, the population will not survive.</p>
<p>Organisms with more DNA generally have more mutations, since the number of errors increases as
	more is copied.&nbsp; If most of the DNA in large genome organisms (e.g. mammals) is functional,
	then these mutations will usually break important functions.&nbsp; Humans get about 100
	mutations per generation.[^moran-2013]&nbsp; </p>
<p>But if most DNA does nothing, or does not have specific information, then most mutations will be
	harmless.&nbsp; Therefore because of this genetic load problem, evolutionary theory both
	predicts and requires only a small amount of DNA in large genome organisms will be functional.
	&nbsp;Writing in 1972, Susumu Ohno was among other early geneticists who recognized this
	problem:</p>
<blockquote><img src="/common/img/portraits/susumu-ohno.jpg?w=100" class="img-left">
	<p>The moment we acquire 10<sup>5</sup><span> gene loci, the overall
		</span><dfn>deleterious</dfn> mutation rate per generation becomes 1.0 which appears
		to represent an unbearably heavy genetic load... Even if an allowance is made for the
		existence in multiplicates of certain genes, it is still concluded that at the most, only 6%
		of our DNA base sequences is utilized as genes[^ohno-1972]</p>
</blockquote>
<p>By "genes," Ohno is referring to functional elements, not only protein coding genes.
	&nbsp;</p>
<p>The fact that a high deleterious mutation rate leads to decline has been tested rigorously using
	computer simulations.&nbsp; In one study with strong natural selection and only 10 harmful
	mutations per generation, fitness continued to decline perpetually.[^gibson-2011]</p>
<div class="highlight">
	<h4>Why Genetic Load Matters</h4>
	<p>If evolution cannot even maintain these large amounts of functional DNA in complex
		organisms, then it could not have created it. Therefore large amounts of functional DNA is
		good evidence evolutionary theory is incorrect.</p>
</div>
<h2>Common Objections</h2>
<p>This section addresses various objections to the idea that most of the human genome is
	functional.</p>
<h3>Objection 1:&nbsp; The C-Value Paradox</h3>
<p>This objection is addressed in a companion article:</p>
<div class="companionArticle"><a href="/articles/biology/functional-dna-predictions/">
	<img src="/articles/biology/c-value-paradox/header-square.jpg?w=96">
	</a>
	<div><a href="/articles/biology/c-value-paradox"><b>The C-Value
		Paradox</b><br></a>Some organisms have many times more DNA than
		others of similar complexity.&nbsp; Some argue that therefore most DNA in most complex
		organisms must be junk, and thus are not designed.&nbsp; This article provides data
		suggesting otherwise.</div>
</div>
<h3>Objection 2:&nbsp; "DNA that can be removed is junk"</h3>
<p>In 2014 when biologist Craig Venter's team built the first artificial yeast chromosome,
	they left out 20% of the DNA.&nbsp; The yeast still functioned just fine:</p>
<blockquote>
	<p>[T]here are over 50,000 base pairs that were either deleted, inserted or changed in that
		chromosome of 250,000 base pairs, and it works."[^callaway-2014]</p>
</blockquote>
<p>However, like many other organisms, the yeast genome is known to be full of redundant fallback
	systems that kick in when primary systems fail.&nbsp; Among yeast protein coding genes, 80% are
	redundant. Physiologist Dennis Noble explains in a recent lecture, at 16:27:</p>
<blockquote>
	<p>"Simply by knocking genes out we don't necessarily reveal function, because the network may
		buffer what is happening. So you may need to do two knockouts or even three before you
		finally get through to the phenotype. ... If one network doesn't succeed in producing a
		component necessary to the functioning of the cell and the organism, then another network is
		used instead. So most knockouts and mutations are buffered by the network.... [at 19:40] Is
		this an unusual result, ... or is it general? This study went through all 6000 genes in the
		organism yeast. knocking them out one by one. 80% of the knockouts were silent. So this
		physiological process of buffering against gene change is general. It's usual in fact. Now
		that doesn't mean to say that these proteins that are made as a consequence of gene
		templates for them don't have a function. Of course they do. If you stress the organism you
		can reveal the function. .. If the organism can't make product X by mechanism A, it makes it
		by mechanism B."[^noble-2012]</p>
</blockquote>
<p>ENCODE, 2014 also remarked about how redundancy can allow functional DNA to be
	disabled or removed without noticing an effect:</p>
<blockquote>
	<p>Loss-of-function tests can also be buffered by functional redundancy, such that double or
		triple disruptions are required for a phenotypic consequence.[^kellis-2014]</p>
</blockquote>
<p>Therefore removing a functional but redundant section of DNA will not have an effect on the
	phenotype.</p>
<p>Another possibility is that the sequences removed were once functional but had previously been
	disabled by mutation in wild-type yeast.</p>
<h3>Objection 3:&nbsp; "DNA not conserved is not functional."</h3>
<p>This objection depends on the assumption that all organisms were created by an unguided process
	of mutation and selection.&nbsp; In other words, if evolution is false then this objection is
	meaningless.&nbsp; An example of such reasoning:</p>
<ol class="compact">
	<li>Humans and mice have about 10% of their DNA in common (conserved).</li>
	<li>Humans and mice evolved from a common ancestor that lived about 80 million years ago.</li>
	<li>Since evolution is a very slow process,[^dodson-1962] [^moran-2014] it could not have
		created much additional function during those 80 million years.</li>
	<li>Therefore not much more than 10% of human and mouse DNA is functional.</li>
</ol>
<p>Functional genome researchers recognize the fault of this reasoning:</p>
<blockquote>
	<p>[R]elative conservation imputes function, but lack of (discernable) conservation imputes
		nothing...  differential expression (including extensive alternative splicing) of RNAs is a 
		far more accurate guide to the functional content of the human genome than logically 
		circular assessments of sequence conservation, or lack thereof.[^mattick-2013]</p>
</blockquote>
<blockquote>
	<p>Functional sequences include but are not limited to sequences under purifying selection
		at the nucleotide level[^niu-2013]</p>
</blockquote>
<blockquote>Since several known functional long ncRNAs, such as Xist and Air, are poorly conserved, 
it is evident that relative lack of conservation does not necessarily signify lack of function.[^dinger-2009]</blockquote>
<h3>Objection 4:&nbsp; "Biochemical activity ≠ function"</h3>
<h4>Promiscuous transcription factors binding to random DNA</h4>
<p>A study in 2013 showed that it's very common for transcription factor proteins to bind to random
	strings of DNA[^white-2013], suggesting it was careless of <dfn>ENCODE</dfn> to use protein
	binding to detect functional DNA.&nbsp; Biochemist and ID critic Larry Moran similarly
	argued:&nbsp; "We also expect that a lot more of the genome will be transcribed on rare
	occasions just because of spurious (accidental) transcription initiation."[^moran-2011]</p>
<p>However a study in 2017 looked at places in DNA where proteins latch on, across 75
	organisms including humans, mice, fruit flies, and yeast:</p>
<blockquote>
	<p>Using in vitro measurements of binding affinities for a large collection of DNA binding
		proteins, in multiple species, we detect a significant global avoidance of weak binding
		sites in genomes.[^long-2016]</p>
</blockquote>
<p>This is significant because:</p>
<blockquote>
	<p>Most DNA binding proteins recognize degenerate patterns; i.e., they can bind strongly
		to tens or hundreds of different possible words and weakly to thousands or
		more.[^long-2016]</p>
</blockquote>
<p>If proteins bind to DNA largely at random then we would expect to see mostly weak binding.
	&nbsp;But we see mostly strong binding.&nbsp; Therefore most DNA-protein binding indicates
	function.</p>
<h4>A cumulative case for function</h4>
<p>However, protein binding and seeing at least 85% of the DNA copied to RNA don't stand alone as
	evidences of function.&nbsp; As mentioned above:</p>
<ol class="compact">
	<li>DNA is usually copied to RNA in precise patterns depending on cell type and
		developmental stage.</li>
	<li>These RNAs are usually transported to specific locations within cells.</li>
	<li>When tested, these RNAs usually affect development or disease.</li>
	<li>Genome-wide association studies (GWAS) find ~95% of disease/trait associated alleles are
		outside protein coding DNA.</li>
</ol>
<p>These four points are incompatible with transcription being the result of random,
	promiscuous binding.&nbsp; Likewise, non-coding DNA researcher John Mattick comments:</p>
<blockquote>
	<p>Assertions that the observed transcription represents random noise... is more opinion
		than fact and difficult to reconcile with the exquisite precision of differential cell- and
		tissue-specific transcription in human cells.[^mattick-2013]</p>
</blockquote>
<h3>Objection 5:&nbsp; "ENCODE Recanted"</h3>
<p>Some argue that in Kellis et al 2014,[^kellis-2014] the <dfn>ENCODE</dfn> team recanted from
	their 2012 assertion that 80% of DNA is functional.&nbsp; However the paper merely lists
	various, often-non-overlapping lower-bound tests for function, some of which are less than 80%.
	&nbsp;For example they write:</p>
<blockquote>
	<p>Genome-wide biochemical studies, including recent reports from ENCODE, have revealed
		pervasive activity over an unexpectedly large fraction of the genome, including noncoding
		and nonconserved regions and repeat elements. Such results greatly increase upper bound
		estimates of candidate functional sequences.[^kellis-2014]</p>
</blockquote>
<p>Likewise figure 2 in the paper shows the large majority of DNA covered with some evidence of
	function.&nbsp; The authors then contrast this with the evidence that if evolutionary theory is
	true, then most DNA cannot be functional.&nbsp; But they do not try to reconcile these two
	contradictory facts. What does the ENCODE team actually believe about function? Biochemist and
	junk DNA proponent Larry Moran commented on a 2015 ENCODE workshop:</p>
<blockquote>
	<p>The overwhelming impression you get from looking at the presentation is that all the
		researchers believe all their data is real and reflects biological function in some way or
		another.[^moran-2017]</p>
</blockquote>
<h3>Objection 6:&nbsp; "Transposons co-opted active DNA"</h3>
<p>About 44% of human DNA is made up of <dfn>transposons</dfn> and transposon-like repetitive DNA
	elements.[^mills-2007]&nbsp; Some have argued that hundreds of millions of years ago, the
	genomes mammal ancestors were much smaller.&nbsp; Over time, transposons duplicated
	themselves throughout mammal genomes.&nbsp; Those that created copies of themselves within
	frequently used regions of DNA copied themselves faster than those that didn't. Therefore these
	transposons are only transcribed in developmental-stage and cell-type specific patterns because
	the DNA regions where they inserted themselves were already specifically regulated.</p>
<p>Yet this idea fails because transcription often begins within transposons themselves.
	&nbsp;The transposons are often <em>providing</em> the transcription start sites, not hijacking
	them:</p>
<blockquote>
	<p>up to 30% of human and mouse <dfn>transcription</dfn> start sites (TSSs) are located in transposable
		elements and that they exhibit clear tissue-specific and developmental stage–restricted
		expression patterns[^fort-2013]</p>
</blockquote>
<blockquote>
	<p>TEs, and in particular ERVs, have contributed hundreds of thousands of novel regulatory
		elements to the primate lineage.[^jacques-2013]</p>
</blockquote>
<p>Moreso, evolutionarily successful <dfn>transposons</dfn> would only need to integrate themselves into regions
 actively transcribed in germline cells (since that DNA is
 inherited), while integrations into actively transcribed regions in
 other cell types wouldn't matter.  Yet we see these sequences being
 transcribed not primarily in regions active in germlince cells, but
 in all cell types.  Thus this is also inconsistent with mosttransposon-like elements in our genomes 
 originating from selfishly copying themselves.</p>

<p>This idea of transposon co-option is also incompatible with other observations discussed above:
	&nbsp;transcripts are usually taken to specific sub-cellular locations, and when tested they
	usually affect development or disease.</p>
<h3>Objection 7:&nbsp; "Repetitive DNA (Transposons / ERVs / etc.) is junk"</h3>
<p>A future article will give an overview of the known functions of various types of DNA elements.
For now:</p>
<blockquote>The full regulatory potential of <dfn>transposon</dfn>-derived sequences is unknown, but appears to include a range of functions.
They are largely <dfn>transcribed</dfn> in a regulated manner, and feature promoters to drive specific expression. 
Important roles have also been demonstrated for Alu-derived RNAs in the regulation of RNA polymerase II during heat shock
 and the regulation of alternative splicing, translation and mRNA stability]. Similarly, it has been shown that 
 transcripts derived from retrotransposons can regulate chromatin structure in transposon-rich regions such as 
 centromeres and neocentromeres, that LINE L1 retrotransposition can mediate somatic mosaicism in 
 neuronal precursor cells, and that the transcription of inverted repeats that serve as boundary elements 
 can influence gene expression. Moreover, a recent study has shown that 6–30% of cap-selected mouse
 and human RNA transcripts initiate within repetitive elements, that approximately 250,000 of these
  transcripts are generally tissue specific, and that transposon-derived sequences located immediately 
  upstream of protein-coding loci frequently function as alternative promoters and/or express non-coding RNAs, 
  identifying some 23,000 candidate regulatory regions derived from retrotransposons. In addition, 
  repetitive sequences may be included as parts of larger transcripts, including many ncRNAs whose functions 
  have been demonstrated, such as Xist, Air, Kcnq1ot1, BORG, DISC2, NTT and Xlisrts, suggesting that these 
  elements may be functional modules common among ncRNAs as well as mRNAs.[^dinger-2009]</blockquote>

<h3>Other Objections</h3>
<p>These objections are less common, so they are covered more briefly:<br></p>
<ol>
	<li><b>"Genome size is negatively correlated with effective population
		size:"</b> Dan Graur raised this objection on slide 46 of a presentation in
		2013[^graur-2013b], citing a 2005 study of ray-finned fish.[^s-2005]&nbsp; This
		correlation would be predicted by selfish gene theory because "evolution can only produce a
		genome devoid of 'junk' if and only if the effective population size is huge and the
		deleterious effects of increasing genome size are considerable"[^graur-2013]. However
		Graur's argument was addressed by T. Ryan Gregory in 2008, who
		showed that fish genome sizes would have been set long before fish were divided
		into their current population sizes:&nbsp;
		<blockquote>it is apparent that genomes reached their current sizes in most fishes long
			before contemporary microsatellite heterozygosities were shaped[^gregory-2008]
			</blockquote>
	</li>
	<li><b>"ENCODE used transcriptionally abnormal cell lines:"</b>&nbsp; In 2013 Dan
		Graur argued "ENCODE used almost exclusively pluripotent stem cells and cancer cells, which
		are known as transcriptionally permissive environments"[^graur-2013]&nbsp; But this
		misconception was corrected by an ENCODE researcher:&nbsp; <br>
		<blockquote><span>The vast majority of the RNAseq was performed in GM12878 or K562 (but we
			looked at a lot of different cell types, </span>
			<a href="http://genome.ucsc.edu/ENCODE/cellTypes.html">see here.</a>.. That assertion is
			completely false and I don't want that misinformation being spread... We used K562
			because it is transcriptionally normal.[^everywhere-2013]
		</blockquote>
		Because of such problems, one researcher remarked of Graur's 2013 paper:&nbsp; &nbsp;
		<blockquote>Graur wrote such a negative paper that it was hard to read... Graur’s criticism
			is so over-the-top that it’s not worthy.[^bhattacharjee-2014]</blockquote>
	</li>
	<li><b>"The Stat3 transcription factor indicates promiscuous protein binding:"</b>
		&nbsp;Dan Graur also made this argument in 2013.[^graur-2013]&nbsp; The problem here is
		that Graur merely assumes the remaining possible binding sites were non-functional, even
		though they were never tested.&nbsp; Among the 14 that were tested, 12 (86% of them) were
		functional.&nbsp; This is particularly notable since Stat3 is less specific than other
		transcription factors in what it binds to:
		<blockquote>
			<p>we scanned the whole mouse genome sequence finding a total of 1,355,858 putative
				binding sites. Such a high number is not unexpected taking into account the loose
				sequence requirements for Stat3 binding.[^vallania-2009]</p>
		</blockquote>
	</li>
</ol>


<h2>Further Reading</h2>
<ol>
	<li>In <a href="http://shapiro.bsd.uchicago.edu/Shapiro&amp;Sternberg.2005.BiolRevs.pdf">Why
		repetitive DNA is essential to genome function</a> (Biol Rev, 2005), James Shapiro and
		Richard Sternberg identify over 80 roles for repetitive sequences.</li>
	<li><a href="http://www.thefreelibrary.com/Micromanagers%3A+new+classes+of+RNAs+emerge+as+key+players+in+the+brain.-a0176480331">Micromanagers:
		new classes of RNAs emerge as key players in the brain</a> (Science News, 2008)
		provides an overview of many known functions of non-coding RNA.</li>
	<li><a href="http://www.reddit.com/r/askscience/comments/znlk6/askscience_special_ama_we_are_the_encyclopedia_of/">Reddit
		AskScience interview with the ENCODE Team</a>.&nbsp;  One researcher wrote:
		<blockquote>That said, I can’t help but notice a trend: over time, 'junk DNA' is
			disappearing. Good riddance: this is just a term for DNA that we don’t have any guesses
			about its function. The more we learn about the genome, the more functions we uncover,
			thus fewer unknowns and a more seemingly 'useful' genome. Where will it end? I have no
			idea, but many people are looking (though more are always needed!).</blockquote>
		Another researcher wrote:
		<blockquote>It's been my experience that anytime we have to quibble over a soft definition,
			we are missing a larger point. A better question would be, what percentage of the DNA
			sequence could be removed with ABSOLUTELY NO effect? I would guess that it's very, very
			small.</blockquote>
		That comment was later deleted.&nbsp; Biochemist and prominant junk DNA proponent Larry Moran
		also became involved in the discussion.
	</li>
	<li>In
		<a href="http://www.reddit.com/r/askscience/comments/1l3zfx/how_come_theres_a_amoeba_with_200_times_larger/cbvsn0n">this
		thread from 2013</a>, an ENCODE scientist defends the ENCODE conclusions in a debate
		with an evolutionary biologist.</li>
	<li><a href="http://sandwalk.blogspot.com/2013/07/five-things-you-should-know-if-you-want.html">Five
		Things You Should Know if you Want to Participate in the Junk DNA Debate</a>, 2013.
		&nbsp;Biochemist and ID critic Larry Moran outlines why he thinks most DNA is true junk.
		&nbsp;Most of his reasons are based on the idea that evolution couldn't have produced
		anything better.</li>
	<li>In December of 2014, well known genomicist and ID critic Dan Graur (cited frequently
		here) wrote
		<a href="http://judgestarling.tumblr.com/post/106416711426/a-christian-creationist-calculates-the-fraction-of"
		>a blog post</a> about an earlier version of this article.</li>
</ol>
<h2>Sources</h2>
<div class="footnotes">
	<ol>
		<li>[^pufferfish]:Fine, Steven.&nbsp;
			"<a href="https://commons.wikimedia.org/wiki/File:Pufferfish_at_the_Audubon_Aquarium_of_the_Americas.jpg">Pufferfish
			at the Audubon Aquarium of the Americas</a>."&nbsp; Wikipedia.&nbsp;
			2016.<small>Wikipedia says the image is licensed under the Creative
			Commons Attribution-Share Alike 4.0 International license.</small></li>
		<li>[^yong-2012]:Yong, Ed.&nbsp;
			"<a href="http://blogs.discovermagazine.com/notrocketscience/2012/09/05/encode-the-rough-guide-to-the-human-genome/">ENCODE:
			the rough guide to the human genome</a>."&nbsp; Discover Magazine.&nbsp; 2012.</li>
		<li>[^graur-2013b]:Graur, Dan.&nbsp; "How to Assemble a Human Genome."&nbsp; 2013.&nbsp;
			Slide 5.
			<small>Mirrors:
			&nbsp;<a href="http://web.archive.org/web/20130719031933/http://twileshare.com/askq">Archive.org</a>
			|&nbsp;
			<a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png">Local
				screenshot</a></small></li>
		<li>[^behe-2010]:Behe, Michael J.&nbsp;
			"<a href="http://www.journals.uchicago.edu/doi/abs/10.1086/656902">Experimental
			evolution, loss-of-function mutations, and "The first rule of adaptive
			evolution</a>."&nbsp; Quarterly Review of Biology.&nbsp; 2010.<br>
			<small>Michael Behe defines a "functional coding element" as "a discrete but
			not necessarily contiguous region of a gene that, by means of its nucleotide sequence,
			influences the production, processing, or biological activity of a particular nucleic
			acid or protein, or its specific binding to another molecule." <br>Mirrors:
			&nbsp;<a href="https://www.lehigh.edu/bio/Faculty/Behe/PDF/QRB_paper.pdf">Michael Behe's
			LeHigh U. Faculty Page</a> |
			<a href="https://www.scribd.com/document/124714881/Behe-Experimental-Evolution-Loss-of-Function-Mutations-and-The-First-Rule-of-Adaptive-Evolution">Scribd</a>
			| <a href="http://archive.is/RHYrp">Archive.is</a>
			</small>
		</li>
		<li>[^eyre-walker-2007]:Eyre-Walker, Adam et al.&nbsp;
			"<a href="http://www.lifesci.susx.ac.uk/home/Adam_Eyre-Walker/Website/Publications_files/EWNRG07.pdf">The
			distribution of fitness effects of new mutations</a>."&nbsp; Nature.&nbsp; 2007.
			<small>The authors state:&nbsp; "relatively few amino-acid-changing
			mutations have effects of greater than 10% in humans, and that most have effects in the
			range of 10<sup>-3</sup> and 10<sup>-1</sup>" Mutations in synonymous regions and
			non-protein coding genes would have even less of an effect. <br>Mirrors:&nbsp;
			<a href="http://web.archive.org/web/20120301193017/http://www.lifesci.susx.ac.uk/home/Adam_Eyre-Walker/Website/Publications_files/EWNRG07.pdf">Archive.org</a>
			| <a href="http://archive.is/S9DqD">Archive.is</a>
			</small></li>
		<li>[^lind-2010]:Lind, Peter A. et al.&nbsp;
			"<a href="http://science.sciencemag.org/content/330/6005/825">Mutational Robustness of
			Ribosomal Protein Genes</a>."&nbsp; Science.&nbsp; 2010. <small>The
			authors tested two ribosomal proteins in <em>salmonella trphimurium</em> and found:
			"most mutations (120 out of 126) are weakly deleterious and the remaining ones are
			potentially neutral."&nbsp; The authors did not detect any beneficial mutations.&nbsp;
			They suggest their study is more sensitive than others because "Mutagenesisstudies of
			single proteins rarely include the use of high-sensitivity assays of fitness and
			analysis of synonymous substitutions (SOM references)... it is conceivable that the
			relatively high frequencies of apparently neutral mutations observed in certain
			experimental systems are mainly a consequence of the limited sensitivity of the assays
			and that the proportion of deleterious mutations is very high even when synonymous
			substitutions are included."&nbsp; </small></li>
		<li>[^lynch-2006]:Lynch, Michael.&nbsp;
			"<a href="https://academic.oup.com/mbe/article/23/2/450/1119102/The-Origins-of-Eukaryotic-Gene-Structure">The
			Origins of Eukaryotic Gene Structure</a>."&nbsp; Mol Bio Evol.&nbsp; 2006.
			<small>Michael Lynch is a highly respected population geneticist.&nbsp;
			He writes:&nbsp; "The neutral (or nearly neutral) theory that emerged from this work
			still enjoys a central place in the field of molecular evolution" and "it is difficult
			to reject the hypothesis that the basic embellishments of the eukaryotic gene originated
			largely as a consequence of nonadaptive processes operating contrary to the expected
			direction of natural selection." <br>Mirrors:&nbsp;
			<a href="http://archive.is/T101Y">Archive.is</a>
			<br>
			</small></li>
		<li>[^myers-2014]:Myers, PZ.&nbsp;
			"<a href="http://scienceblogs.com/pharyngula/2014/02/14/the-state-of-modern-evolutionary-theory-may-not-be-what-you-think-it-is/">The
			state of modern evolutionary theory may not be what you think it is</a>."&nbsp;
			Pharyngula Blog.&nbsp; 2014. <small>PZ Myers is a developmental
			biologist well known for his criticisms of Intelligent Design.&nbsp; Myers writes:&nbsp;
			"the revolution is over. Neutral and nearly neutral theory won." <br>Mirrors:
			&nbsp;<a href="http://archive.is/XXNPj">Archive.is</a>
			<br>
			</small></li>
		<li>[^dodson-1962]:Dodson, Edward O.&nbsp;
			"<a href="http://www.journals.uchicago.edu/doi/abs/10.1086/282213">Note on the Cost of
			Natural Selection</a>."&nbsp; The American Naturalist.&nbsp; 1962.
			<small>JBS Haldane was an evolutionary biologist well known for his work
			in developing the modern evolutionary synthesis.&nbsp; Calculating the implications of
			Haldane's model, Dodson explains: "Haldane (1957) has published calculations which
			indicate that it takes no less than 300 generations to replace a gene by ordinary
			selection pressures, and that this evolutionary process cannot be speeded up by
			simultaneous selection for more than one gene....we arrive at a maximum of something
			over 200 gene substitutions over the past million years for the genus Homo."&nbsp; Since
			on average the mutation rate is the fixation rate, 300 generations with about 100
			mutations per generation would give 30,000 neutral mutations per 1 beneficial mutation
			that fixes.&nbsp; 29,999 / 30,000 is <b>99.997%</b> of fixed mutatiosn being
			neutral.<br>Mirrors:&nbsp;
			<a href="functional-dna/sources/dodson-1962.png">Local screenshot</a>
			</small></li>
		<li>[^moran-2014]:Moran, Larry.&nbsp; Comment on
			"<a href="http://sandwalk.blogspot.com/2014/04/breaking-news-creationist-vincent.html?showComment=1396459508015#c3928367699865279715">Breaking
			news: Creationist Vincent Torley lies and moves goalposts</a>."&nbsp; Sandwalk
			Blog.&nbsp; 2014. <small>Joe Felsenstein is a well known population
			geneticist who has published a
			<a href="https://www.jstor.org/stable/2459384?seq=1">criticism</a> of
			Haldane's limit.&nbsp; When he and biochemist Larry Moran (both Intelligent Design
			critics) were asked to estimate the number of beneficial versus neutral differences
			between human and chimps, they replied:&nbsp; "Updated numbers suggest 44 million point
			mutations and something like 2 million insertions/deletions for a grand total of 46
			million mutations.&nbsp; We don't know how many of those were beneficial (adaptive)
			leading to ways in which modern chimps are better adapted than the common ancestor.
			(Same for humans.) My guess would be only a few thousand in each lineage."&nbsp;  1 -
			3000 / 23,000,000 is <b>99.987%</b> of fixed mutations being
			neutral.</small></li>
		<li>[^meader-2010]:Meader et al.&nbsp;
			"<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2945182/">Massive turnover of
			functional sequence in human and other mammalian genomes</a>."&nbsp; Genome Res.&nbsp;
			2010. <small>Figure 2 shows how much DNA (of 3 billion base pairs total)
			is shared between humans (homo), macaca monkeys, mice (mus), rats (rattus), horses
			(equus), cows (bos), and dogs (canis). <br>Mirrors:
			&nbsp;<a href="http://archive.is/PNVyB">Archive.is</a>
			</small></li>
		<li>[^graur-2013]:Graur, Dan.&nbsp;
			"<a href="http://gbe.oxfordjournals.org/content/early/2013/02/20/gbe.evt028.full.pdf">On
			the Immortality of Television Sets:&nbsp; 'Function' in the human genome according to
			the evolution-free gospel of ENCODE</a>."&nbsp; Genome Biology and Evolution.&nbsp;
			2013.<small>Mirrors:
			&nbsp;<a href="https://web.archive.org/web/20160622030232/http://gbe.oxfordjournals.org/content/early/2013/02/20/gbe.evt028.full.pdf">Archive.org</a>
			| <a href="functional-dna/sources/graur-function-in-the-human-genome-2013">Local
			excerpt with notes</a></small></li>
		<li>[^hangauer-2013]:Hangauer et. al.&nbsp;
			"<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003569">Pervasive
			Transcription of the Human Genome Produces Thousands of Previously Unidentified Long
			Intergenic Noncoding RNAs</a>."&nbsp; PLoS Genetics.&nbsp; 2013.
			<small>Mirrors:
			&nbsp;<a href="http://archive.is/VLHiU">Archive.is</a><a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png"></a>
			|
			<a href="functional-dna/sources/hangauer-pervasive-transcription-2013">Local
			excerpt with notes</a>
			</small></li>
		<li>[^kellis-2014]:Kellis et.al.&nbsp;
			"<a href="http://www.pnas.org/content/111/17/6131.long">Defining functional DNA elements
			in the human genome</a>."&nbsp; PNAS.&nbsp; 2014. <small>Figures #1 and
			#2 show detected functional DNA by different methodologies.<br>Mirrors:
			&nbsp;<a href="http://archive.is/VLHiU">Archive.is</a><a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png"></a>
			|
			<a href="functional-dna/sources/kellis-defining-functional-dna-2014">Local
			excerpt with notes</a></small></li>
		<li>[^encode-2012]:The ENCODE Project Consortium.&nbsp;
			"<a href="http://www.nature.com/nature/journal/v489/n7414/full/nature11247.html?WT.ec_id=NATURE-20120906">An
			integrated encyclopedia of DNA elements in the human genome</a>."&nbsp; Nature.&nbsp;
			2012.</li>
		<li>[^mattick-2010]:Mattick, John.&nbsp;
			"<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2905358/">Video Q&amp;A:
			Non-coding RNAs and eukaryotic evolution - a personal view</a>."&nbsp; BMC Biol.&nbsp;
			2010.</li>
		<li>[^mattick-2013]:Mattick, John S. and Marcel E. Dinger.&nbsp;
			"<a href="https://thehugojournal.springeropen.com/articles/10.1186/1877-6566-7-2">The
			extent of functionality in the human genome</a>."&nbsp; HUGO.&nbsp; 2013.
			<small>Mirrors:
			&nbsp;<a href="http://archive.is/ONipd">Archive.is</a><a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png">
			| </a><a href="/articles/reviews/mattick-extent-of-functionality-2013.php">Local
			excerpt with notes</a>
			</small></li>
		<li>[^garvan-2013]:"<a href="https://www.garvan.org.au/news-events/news/new-insight-into-the-human-genome-through-the-lens-of-evolution">New
			insight into the human genome through the lens of evolution</a>."&nbsp; Garvan
			Institute.&nbsp; 2013. <small>Mirrors:&nbsp;
			<a href="http://archive.is/TWtUu">Archive.is</a>
			<a href="https://web.archive.org/web/20160106070951/http://www.garvan.org.au/news-events/news/new-insight-into-the-human-genome-through-the-lens-of-evolution">Archive.org</a>
			</small></li>
		<li>[^parker-2011]:"<a href="http://genome.cshlp.org/content/early/2011/10/03/gr.112516.110.full.pdf">New
			families of human regulatory RNA structures identified by comparative analysis of
			vertebrate genomes</a>."&nbsp; Genome Research.&nbsp; 2011.<small>See
			the diagrams of functional RNAs in figure 5.&nbsp; 66% of nucleotides are cross linked
			and thus at least that much has a specific sequence.<br>Mirrors:
			&nbsp;<a href="http://archive.is/Hew2l">Archive.is</a><a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png">
			</a></small></li>
		<li>[^frankel-2014]:Frankel, Lisa B et al.&nbsp;
			"<a href="https://www.nature.com/articles/ncomms6840">A non-conserved miRNA regulates
			lysosomal function and impacts on a human lysosomal storage disorder</a>."&nbsp;
			Nature.&nbsp; 2014.<small>Mirrors:
			&nbsp;<a href="http://archive.is/njTSp">Archive.is</a><a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png">
			</a></small></li>
		<li>[^hsa-mir-95]:"<a href="http://mirnamap.mbc.nctu.edu.tw/php/mirna_entry.php?acc=MI0000097">The
			pre-miRNA of MI0000097</a>."&nbsp; miRNAMap. <small>Mirrors:
			&nbsp;<a href="http://archive.is/omyS6">Archive.is</a>
			<a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png">
			</a>
			</small></li>
		<li>[^birney-2012]:Birney, Ewan.&nbsp;
			"<a href="http://genomeinformatician.blogspot.com/2012/09/encode-my-own-thoughts.html">ENCODE:
			My own thoughts</a>."&nbsp; Ewan's Blog.&nbsp; 2012.</li>
		<li>[^maurano-2012]:Maurano, Matthew T.&nbsp;
			"<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3771521/bin/NIHMS492174-supplement-SOM.pdf">Supplementary
			Materials for Systematic Localization of Common Disease Associated Variation in
			Regulatory DNA</a>."&nbsp; Science.&nbsp; 2012.<small>See figure S1 for the
			pie chart of SNP locations.&nbsp; The caption states "only 4.9% of GWAS SNPs lie 
			in coding sequence."&nbsp; The study says "coding regions were defined by
			the CCDS Project (downloaded&nbsp; from&nbsp; the UCSC genome&nbsp; browser at
			http://hgdownload.cse.ucsc.edu/goldenPath/hg19/database/ccdsGene.txt.gz on March
			5, 2011"</small></li>
		<li>[^freedman-2011]:Freedman, Matthew L.&nbsp;
			"<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3325768/">Principles for the
			post-GWAS functional characterization of cancer risk loci</a>."&nbsp; Nature
			Genetics.&nbsp; 2011. <small>See table 1 in the paper.&nbsp; Note that a
			pre-print version of this study listing Alvaro N.A. Monteiro as the lead author does not
			include the table. <br>Mirrors: <a href="sources/freedman-2012-table1.png">local
			screenshot</a> | <a href="http://archive.is/85kcr">Archive.is</a>
			</small></li>
		<li>[^mattick-2009]:Mattick, John S.&nbsp;
			"<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000459">The
			Genetic Signatures of Noncoding RNAs</a>."&nbsp; PLOS Genetics.&nbsp; 2009.
			<small>The author states:&nbsp; "Most variations in regulatory sequences
			produce relatively subtle phenotypic changes, in contrast to mutations in protein-coding
			sequences that frequently cause catastrophic component failure." <br>Mirrors:
			&nbsp;<a href="http://archive.is/omyS6">Archive.is</a>
			<a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png">
			</a>
			</small></li>
		<li>[^chen-2015]:Chen, Geng.&nbsp;
			"<a href="https://www.nature.com/articles/srep09453">Re-annotation
			of presumed noncoding disease/trait-associated genetic variants by integrative
			analyses</a>."&nbsp; Scientific Reports.&nbsp; 2015. <small>Mirrors:
			&nbsp;<a href="http://archive.is/bhdRf">Archive.is</a>
			<a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png">
			</a>
			</small></li>
		<li>[^smith-2013]:Smith, Martin et al.&nbsp; "Widespread purifying selection on RNA
			structure in mammals."&nbsp; Nucleic Acid Research.&nbsp; 2013.
			<small>Mirrors:
			&nbsp;<a href="http://archive.is/ZuR7p">Archive.is</a>
			<a href="functional-dna/sources/graur-how-to-assemble-a-human-genome-2013-slide5.png">
			|
			</a><a href="/articles/reviews/smith-widespread-purifying-selection-on-rna-2013.php">Local
			excerpt with notes</a>
			</small></li>
		<li>[^rands-2014]:Rands, Chris M. et al.&nbsp;
			"<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004525">8.2%
			of the Human Genome Is Constrained</a>."&nbsp; PLOS Genetics.&nbsp; 2014.</li>
		<li>[^preuss-2012]:Preuss, Todd M.&nbsp;
			"<a href="http://www.pnas.org/content/109/Supplement_1/10709.full.pdf">Human brain
			evolution: From gene discovery to phenotype discovery</a>."&nbsp; PNAS.&nbsp; 2012.<br></li>
		<li>[^axe-2004]:Axe, Doug.&nbsp; "<a href="http://ge.tt/18e1v2K/v/0">Estimating the
			Prevalence of Protein Sequences Adopting Functional Enzyme Folds</a>."&nbsp; J. Mol
			Biol.&nbsp; 2004. Doug Axe (an ID proponent) tested a protein with degraded function to
			see how many amino acids could be replaced:&nbsp; "about one in four random
			single-residue [amino acid] changes are functionally neutral. The proportion would be
			somewhat lower under conditions requiring a higher level of function."</li>
		<li>[^vallania-2009]:Vallania, Francesco et al.&nbsp;
			"<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2664024/">Genome-wide discovery
			of functional transcription factor binding sites by comparative genomics: The case of
			Stat3</a>."&nbsp; PNAS.&nbsp; 2009.</li>
		<li>[^white-2013]:White, Michael A.&nbsp;
			"<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3718143/">Massively parallel in
			vivo enhancer assay reveals that highly local features determine the cis-regulatory
			function of ChIP-seq peaks</a>."&nbsp; PNAS.&nbsp; 2013. <small>The lead
			author summarizes the paper
			<a href="https://thefinchandpea.com/2013/07/17/using-a-null-hypothesis-to-find-function-in-the-genome/">on
			his blog</a>.&nbsp; (<a href="http://archive.is/TUPpu">Archive.is</a>).&nbsp; He also
			<a href="http://www.homolog.us/blogs/blog/2013/07/17/random-dna-sequence-mimics-encode/">tweeted</a>:&nbsp;
			"One of our Nature reviewers said this paper should not be published in any journal,
			ever."</small></li>
		<li>[^long-2016]:Qian, Long and Edo Kussel.&nbsp;
			"<a href="https://journals.aps.org/prx/pdf/10.1103/PhysRevX.6.041009">Genome-Wide Motif
			Statistics are Shaped by DNA Binding Proteins over Evolutionary Time Scales</a>."&nbsp;
			Physical Review X.&nbsp; 2016.</li>
		<li>[^callaway-2014]:Calloway, Ewen.&nbsp;
			"<a href="http://www.nature.com/news/first-synthetic-yeast-chromosome-revealed-1.14941">First
			synthetic yeast chromosome revealed</a>."&nbsp; Nature News.&nbsp; 2014.<br></li>
		<li>[^noble-2012]:Noble, Dennis.&nbsp;
			"<a href="https://www.youtube.com/watch?v=QMVfafAYTMg">Physiology and
			Evolution</a>."&nbsp; 2012.&nbsp; At
			<a href="https://www.youtube.com/watch?v=QMVfafAYTMg&amp;t=16m24s">16:24</a>.</li>
		<li>[^niu-2013]:Niu, Deng-ke.&nbsp;
			"<a href="http://www.sciencedirect.com/science/article/pii/S0006291X12024229">Can ENCODE
			tell us how much junk DNA we carry in our genome?</a>"&nbsp; Biochemical and
			Biophysical Research Communications.&nbsp; 2013.</li>
		<li>[^moran-2017]:Moran, Larry.&nbsp;
			"<a href="http://sandwalk.blogspot.com/2017/02/encode-workshop-discusses-function-in.html">ENCODE
			workshop discusses function in 2015</a>."&nbsp; Sandwalk Blog.&nbsp; 2017.</li>
		<li>[^everywhere-2013]:Reddit user west_of_everywhere.&nbsp;
			"<a href="https://www.reddit.com/r/askscience/comments/1l3zfx/how_come_theres_a_amoeba_with_200_times_larger/cbw1bze/">Comment
			on How come there's a Amoeba with 200 times larger gene set than humans?</a>"&nbsp;
			Reddit.&nbsp; 2013.</li>
		<li>[^bhattacharjee-2014]:Bhattacharjee, Yudhijit.&nbsp;
			"<a href="http://science.sciencemag.org/content/343/6177/1306">The Vigilante</a>
			."&nbsp; Science.&nbsp; 2014.</li>
		<li>[^s-2005]:S, Yi.&nbsp; "<a href="https://www.ncbi.nlm.nih.gov/pubmed/16213058">Genome
			size is negatively correlated with effective population size in ray-finned
			fish</a>."&nbsp; Trends Genet.&nbsp; 2005.</li>
		<li>[^gregory-2008]:Gregory, T. Ryan.&nbsp;
			"<a href="https://www.ncbi.nlm.nih.gov/pubmed/18356967">Population size and genome size
			in fishes: a closer look</a>."&nbsp; Genome.&nbsp; 2008.</li>
		<li>[^mills-2007]:Mills.&nbsp; Ryan E et al.&nbsp;
			"<a href="http://www.cell.com/trends/genetics/fulltext/S0168-9525(07)00059-5">Which
			transposable elements are active in the human genome?</a>"&nbsp; Trends in
			Genetics.&nbsp; 2007.</li>
		<li>[^fort-2013]:Fort, Alexandre et al.&nbsp;
			"<a href="http://www.nature.com/ng/journal/v46/n6/full/ng.2965.html">Deep transcriptome
			profiling of mammalian stem cells supports a regulatory role for retrotransposons in
			pluripotency maintenance</a>."&nbsp; Nature Genetics.&nbsp;
			2013.<small>Mirrors:&nbsp;
			<a href="https://www.ohio.edu/bioinformatics/upload/ng-2965.pdf">Ohio.edu</a> |
			<a href="sources/fort-retrotransposons-2013.png">Local screenshot</a></small></li>
		<li>[^jacques-2013]:Jacques, Pierre-Étienne et al.&nbsp;
			"<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003504">The
			Majority of Primate-Specific Regulatory Sequences Are Derived from Transposable
			Elements</a>."&nbsp; PLOS Genetics.&nbsp; 2013.</li>
		<li>[^moran-2011]:Moran, Larry.&nbsp;
			"<a href="http://sandwalk.blogspot.com/2011/05/junk-jonathan-part-6-3.html">Junk &amp;
			Jonathan: Part 6—Chapter 3</a>."&nbsp; Sandwalk Blog. 2011.</li>
		<li>[^gibson-2011]:Gibson et al.&nbsp;
			"<a href="http://www.worldscientific.com/doi/pdf/10.1142/9789814508728_0010">Can
			Purifying Natural Selection Preserve Biological Information?</a>"&nbsp; World
			Scientific.&nbsp; 2011.</li>
		<li>[^ohno-1972]:Ohno, Susumu.&nbsp; "<a href="http://www.junkdna.com/ohno.html">So much
			'Junk' in our Genome</a>."&nbsp; Brookhaven Symposia in Biology.&nbsp; 1972. <br>
			<small>Mirrors:
			&nbsp;<a href="https://web.archive.org/web/20160203012605/http://www.junkdna.com/ohno.html">Archive.org</a>
			| <a href="http://archive.is/j22Qx#selection-79.366-79.535">Archive.is</a> |
			<a href="/articles/reviews/ohno-so-much-junk-in-our-genome-1972-comment.php">Local
			excerpt with comment</a></small>
		</li>
		<li>[^moran-2013]:Moran, Larry.&nbsp;
			"<a href="http://sandwalk.blogspot.com/2013/03/estimating-human-mutation-rate-direct.html">Estimating
			the Human Mutation Rate: Direct Method</a>."&nbsp; Sandwalk Blog.&nbsp; 2013.</li>
		<li>[^moran-2014b]:Moran, Larry.&nbsp; 
			"<a href="http://sandwalk.blogspot.com/2014/04/a-creationist-tries-to-understand.html">A
			creationist tries to understand genetic load</a>."&nbsp; Sandwalk Blog.&nbsp; 
			2014.<small>Mirrors:&nbsp;
			 <a href="http://web.archive.org/web/20150906213536/http://sandwalk.blogspot.com/2014/04/a-creationist-tries-to-understand.html">Archive.org</a> |
			<a href="http://archive.is/MrGhZ">Archive.is</a></small></li>
		<li>[^dna-transcription-2010]:DNA Learning Center.&nbsp; "DNA Transcription (Basic)."&nbsp;
			YouTube.&nbsp; 2010.</li>
			
		<li>[^khaitan-2011]:Divya Khaitan et al.&nbsp; "<a href="http://cancerres.aacrjournals.org/content/71/11/3852">The
		Melanoma‐Upregulated Long Noncoding RNA SPRY4-IT1 Modulates Apoptosis and Invasion.</a>"&nbsp; Molecular and Cellular Pathobiology.&nbsp; 2011.</li>
		<li>[^guttman-2010]:Mitchell Guttman et al.&nbsp; "<a href="https://www.nature.com/articles/nature10398">lincRNAs
		act in the circuitry controlling pluripotency and differentiation</a>."&nbsp; Nature.&nbsp; 2011.</li>
		<li>[^dinger-2009]:MArcel E. Dinger et al.&nbsp; "<a href="https://academic.oup.com/bfg/article/8/6/407/216134">Pervasive
		transcription of the eukaryotic genome: functional indices and conceptual implications</a>."&nbsp; Briefings in Functional Genomics.&nbsp; 2009.</li>
		<li>[^ng-2012]:Shi-Yan Ng et al.&nbsp; "<a href="http://emboj.embopress.org/content/31/3/522">Human long non‐coding RNAs promote pluripotency and neuronal differentiation by association with chromatin modifiers and transcription factors</a>."&nbsp; EMBO.&nbsp; 2012.</li>
		<li>[^sunwoo-2009]:Hongjae Sunwoo et al.&nbsp; "<a href="https://genome.cshlp.org/content/19/3/347">MEN ε/β nuclear-retained non-coding RNAs are up-regulated upon muscle differentiation and are essential components of paraspeckles</a>."&nbsp; Genome Research.&nbsp; 2009.</li>
		<li>[^askarian-amiri-2011]:Marjan E. Askarian-Amiri et al.&nbsp; "<a href="http://rnajournal.cshlp.org/content/17/5/878">SNORD-host RNA Zfas1 is a regulator of mammary development and a potential marker for breast cancer</a>."&nbsp; RNA.&nbsp; 2011</li>
	</ol>
</div>
<?php $content = ob_get_clean();


require_once 'common/includes/theme.php';
